pgcli/pgexecute.py

```diff
@@ -11,6 +12,9 @@
 psycopg2.extensions.register_type(psycopg2.extensions.UNICODE)
 psycopg2.extensions.register_type(psycopg2.extensions.UNICODEARRAY)
 
+# When running a query, make pressing CTRL+C raise a KeyboardInterrupt
+psycopg2.extensions.set_wait_callback(psycopg2.extras.wait_select)
```

Comment:
This is interesting. It works as expected but I don't understand how. Reading the documentation says this waits for data to be available, but during the wait time does it automatically capture Ctrl+C and convert it to KeyboardInterrupt? Is this a behavior of `select()` or `wait_select()`?